Docs: Windows CPU and Memory Affinity (#48469)
kubernetes/website: content/en/docs/tasks/administer-cluster/memory-manager.md

in order to enable the Memory Manager feature.

> ## How does the Memory Manager Operates?

Reviewer: ?

Reviewer 2: IMO we can keep this line as is prior to the PR

Reviewer 3: Or it can be:
```suggestion
## How does the Memory Manager Operate?
```

Author: I took the opportunity to improve the grammar here but happy to remove it if its to much noise.

Also, reference [Memory Manager KEP: Simulation - how the Memory Manager works? (by examples)][1]
illustrates how the management of groups occurs.

### Windows Support

> {{< feature-state for_k8s_version="v1.32" state="alpha" feature_gate_name="WindowsCPUAndMemoryAffinity" >}}

Reviewer: ```suggestion
{{< feature-state feature_gate_name="WindowsCPUAndMemoryAffinity" >}}
```

> Windows support can be enabled via the `WindowsCPUAndMemoryAffinity` feature gate and requires a support in the container runtime.  Only the [BestEffort Policy](#policy-best-effort) is supported on Windows.

Reviewer: ```suggestion
Windows support can be enabled via the `WindowsCPUAndMemoryAffinity` feature gate and requires a support in the container runtime.
Only the [BestEffort Policy](#policy-best-effort) is supported on Windows.
```

Also, reference [Memory Manager KEP: Simulation - how the Memory Manager works? (by examples)][1]
illustrates how the management of groups occurs.

### Windows Support

{{< feature-state feature_gate_name="WindowsCPUAndMemoryAffinity" >}}

> Windows support can be enabled via the `WindowsCPUAndMemoryAffinity` feature gate and requires support in the container runtime.

Reviewer: ```suggestion
Windows support can be enabled via the `WindowsCPUAndMemoryAffinity` feature gate
and it requires support in the container runtime.
```

the default topology hint as there is no request for the guaranteed memory,
and does not reserve the memory in the internal [NodeMap][2] object.

> This policy is only Supported on Linux.

Reviewer: ```suggestion
This policy is only supported on Linux.
```

#### BestEffort policy {#policy-best-effort}

> {{< feature-state for_k8s_version="v1.32" state="alpha" feature_gate_name="WindowsCPUAndMemoryAffinity" >}}

Reviewer: ```suggestion
{{< feature-state feature_gate_name="WindowsCPUAndMemoryAffinity" >}}
```

> This policy is only Support on Windows.

Reviewer: ```suggestion
This policy is only supported on Windows.
```

> On Windows, NUMA node assignment works differently than Linux.  There is no mechanism

Reviewer: ```suggestion
On Windows, NUMA node assignment works differently from Linux.  There is no mechanism
```

> to ensure that Memory access only comes from a specific NUMA node, instead the Windows

Reviewer: ```suggestion
to ensure that memory access only comes from a specific NUMA node, instead the Windows
```

> internals will select the most optimal NUMA node based on the CPU assignment and it is

Reviewer: ```suggestion
kernel will select the most optimal NUMA node based on the CPU assignment and it is
```

Reviewer 2: replace CPU with CPU(s) ?

Reviewer 3: ```suggestion
scheduler will select the most optimal NUMA node based on the CPU assignments and it is
```

possible that Windows might use other nodes if required or deemed optimal.  For this
reason this policy is best effort.  

The policy does track the amount of memory available and requested through the internal
[NodeMap][2] to make a best effort at ensuring that memory enough memory is available on

> a NUMA node before making the hint to assign it to the NODE.  This means that in most cases, this should function as expected.

Reviewer: Please consider revise the looooooooong sentence into shorter ones, in simple English.
The text is for user to understand anyway.

Author: I've attempted to simplify with shorter sentences.  PTAL
